src/Symfony/Component/Validator/Constraints/Range.php

```diff
@@ -36,14 +38,30 @@ class Range extends Constraint
     public $maxMessage = 'This value should be {{ limit }} or less.';
     public $invalidMessage = 'This value should be a valid number.';
     public $min;
+    public $minPropertyPath;
     public $max;
+    public $maxPropertyPath;
 
     public function __construct($options = null)
     {
+        if (\is_array($options)) {
+            if (isset($options['min']) && isset($options['minPropertyPath'])) {
```

Comment:
Would it make sense to not introduce new options, but just interpret the existing `min` and `max` options as property paths instead when they are strings?

Reply:
It would not be consistent with what has been done for the comparison constraints. But I like the idea otherwise.

Reply:
There it was not possible. But here it is because of the types currently support by `min` and `max`.

Reply:
@xabbuh min/max already support string values ...

> type: number or string (date format)

Reply:
Oh, I stand corrected then. Thanks for noticing @ro0NL.